dss/operations/util.py

```diff
@@ -39,3 +44,137 @@ def map_bucket_results(func: typing.Callable, handle: BlobStore, bucket: str, ba
 def map_bucket(*args, **kwargs):
     for _ in map_bucket_results(*args, **kwargs):
         pass
+
+def set_cloud_env_var(
+    environment: dict,
+    env_set_fn,
+    env_var: str,
+    value,
+    where: str = "",
+) -> None:
+    """
+    Set a variable in a cloud environment by calling env_set_fn.
+    (We pass the environment in directly, instead of passing
+    a get_fn, because the get_fn may require input arguments.)
+
+    Args:
+        environment: A dictionary containing all environment variables.
+        env_set_fn: A function handle used to set a new environment.
+        env_var: The name of the environment variable to set.
+        val: The value of the environment variable to set.
+        where: A label for telling the user where the env var was set.
+    """
+    environment[env_var] = value
+    env_set_fn(environment)
+    print(f'Set variable "{env_var}" {where}')
+
+def unset_cloud_env_var(
+    environment: dict,
+    env_set_fn,
+    env_var: str,
+    where: str = "",
+) -> None:
+    """
+    Unset a variable in an environment by calling env_set_fn.
+
+    Args:
+        environment: A dictionary containing all environment variables.
+        env_set_fn: A function handle used to set a new environment.
+        env_var: The name of the environment variable to unset.
+        where: A label for telling the user where the env var was unset.
+    """
+    try:
+        del environment[env_var]
+        env_set_fn(environment)
+        print(f'Unset variable "{env_var}" {where}')
+    except KeyError:
+        print(f'Nothing to unset for variable "{env_var}" {where}')
+
+def get_cloud_variable_prefix() -> str:
```

Comment:
The environment variables `DSS_SECRETS_STORE` and `DSS_PARAMETER_STORE` have the same value by coincidence. If they diverge, the SSM ops scripting will break.

Please compose the SSM and Secretsmanager store names in their respective operational scripting files.

Reply:
I have moved these methods (to get and fix variable prefixes) to their respective modules in `dss/operations`. In `params.py` we have `get/fix_ssm_variable_prefix()` and in `secrets.py` we have `get/fix_secret_variable_prefix()`. 
* https://github.com/HumanCellAtlas/data-store/blob/dss-params/dss/operations/params.py#L26-L39
* https://github.com/HumanCellAtlas/data-store/blob/dss-params/dss/operations/secrets.py#L52-L65